pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: Dart Team <misc@dartlang.org>
 description: A documentation generator for Dart.
 homepage: https://github.com/dart-lang/dartdoc
 environment:
-  sdk: '>=1.9.0 <2.0.0'
+  sdk: '>=1.11.0 <2.0.0'
```

Comment:
also, it looks like the lower bounds will need to be a dev build - something like `1.11.0-dev.5.2`